.github/workflows/ci.yml

```diff
@@ -132,7 +132,7 @@ jobs:
 
   build_test_all_macos_arm64:
     needs: setup
-    if: fromJson(needs.setup.outputs.should-run) && ! fromJson(needs.setup.outputs.is-pr)
+    if: fromJson(needs.setup.outputs.should-run)
     runs-on:
       - ${{ github.repository == 'openxla/iree' && 'self-hosted' || 'macos-11' }} # must come first
```

Comment:
This might get in the way of running on presubmit. Maybe create your branch in the core repo and not your fork.

Reply:
I see, continuing in #13697 then.